### PR TITLE
add main content

### PR DESCRIPTION
**Pull request by shivamSspirit**

### CODE REVIEW

File: Components/document-content/index.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+}
+
+.aside {
+	background-color: var(--default-color);
+	grid-area: aside;
+	color: black;
+	overflow-y: hidden;
+}
+
+.sidebar {
```
Comment (poojavishnoi) [inline Components/document-content/index.css]: you can remove this.

File: Components/document-content/index.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>main content</title>
+	<link rel="stylesheet" href="./index.css" />
```
Comment (poojavishnoi) [inline Components/document-content/index.html:9]: You can add meta tag for author and description.

File: Components/document-content/index.html [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+		
+			<iframe src="../iframe/iframe.html" name="iframe_a" height="100%" width="100%" title="Iframe Example" style="border:none;"> 
+			</iframe>
+		
+			
+		</main>
+		<!-- <footer class="footer">Footer</footer> -->
+	</section>
+	<!-- script for index -->
+	<script type="text/javascript">
```
Comment (poojavishnoi) [inline Components/document-content/index.html:86]: you can separate the js files.

**Review by poojavishnoi — COMMENTED** [review]
Added few suggestions.

File: Components/document-content/index.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+}
+
+@media only screen and (max-width: 500px) {
+	#container {
+		grid-template-columns: 2fr;
+		grid-template-areas:
+			"header"
+			"aside"
+			"main";
+	}
```
Comment (codewithrishu) [inline Components/document-content/index.css]: try using media queries in the end

File: Components/document-content/index.css [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+.getting-started {
+	list-style: none;
+}
+
+.p0-screens {
+	list-style: none;
+}
+
+.p1-screens {
+	list-style: none;
```
Comment (codewithrishu) [inline Components/document-content/index.css:91]: you can combine these three

File: Components/document-content/index.css [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+.heading {
+	font-size: var(--main-head-size);
+	font-weight: var(--main-head-weight);
+	color: var(--main-text-color);
+}
+
+/* .footer {
+	background-color: var(--default-color);
+	grid-area: footer;
+} */
```
Comment (codewithrishu) [inline Components/document-content/index.css]: unwanted code

File: Components/document-content/index.html [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+		src="https://cdn.jsdelivr.net/gh/google/code-prettify@master/loader/run_prettify.js?autoload=true&amp;skin=sunburst&amp;lang=css"
+		defer=""></script>
+</head>
+
+<body>
+	<section id="container">
+		<header class="header">
+			<div class="icon-container">
+				<a class="img-link" href="../../index.html"><span>
+						<img class="icon" src="../../images/jpeg/headicon.png" alt="" /></a>
```
Comment (codewithrishu) [inline Components/document-content/index.html:20]: never leave alt tag empty

File: Components/document-content/index.html [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+				<a href="../../index.html" class="home navs">Home</a>
+				<a href="./index.html" class="document navs">Document</a>
+			</div>
+			<div class="socials">
+				<span><a href="https://github.com/shivamsoni00"><img src='../../images/jpeg/github.png'
+							class="github" /></a></span>
+				<span><a href="https://www.linkedin.com/in/shivam-s-b37b00157/"><img src='../../images/jpeg/linkedin.png'
+							class="linkedin" /></a></span>
+				<span><a href="https://twitter.com/ShivamSspirit">
+						<img src='../../images/jpeg/twitter.png' class="twitter" /></span>
```
Comment (codewithrishu) [inline Components/document-content/index.html]: insert alt tag

**Review by codewithrishu — APPROVED** [review]

**Pull request merged**